src/Illuminate/Mail/Mailer.php

```diff
@@ -304,18 +302,6 @@ protected function getQueuedCallable(array $data)
     }
 
     /**
-     * Force the transport to re-connect.
-     *
-     * This will prevent errors in daemon queue situations.
-     *
-     * @return void
-     */
-    protected function forceReconnection()
```

Comment:
But it is not used anywhere else but I sure could revert that change if you insist.